packages/wdio-cli/src/commands/config.ts

```diff
@@ -78,8 +101,9 @@ const runConfig = async function (useYarn: boolean, yes: boolean, exit = false)
                 }
             }
 
+            fs.ensureDirSync(path.join(process.cwd(), 'test'))
```

Comment:
if we can use `await`, we should use it
```suggestion
            await fs.ensureDir(path.join(process.cwd(), 'test'))
```